l/libgcrypt/stone.yaml

```diff
@@ -17,9 +17,13 @@ license     :
     - LGPL-2.1-or-later
 builddeps   :
     - pkgconfig(gpg-error)
+checkdeps   :
```

Comment:
Allowing for now but obviously in time for multiarch stuff need a generic approach (autofill architecture for isa)